### ISSUE TITLE
I18n: Use Glotpress for translations

### ISSUE THREAD

**Issue by JoryHogeveen** [pods-framework/pods#3725]
- [ ] Put all translations into translate.wordpress.org/projects/wp-plugins/pods
- [ ] Call for translation editors for various languages
- [ ] Remove translations that are 100% from the Pods plugin
- [ ] Remove auto updates for .po and .mo files from wp.org when ready (#2633)
- [ ] Auto update `trunk` on wp.org with the latest dev from the main GH branche

**Comment by sc0ttkclark** [comment]
I thought we had no languages in the plugin already?

**Comment by JoryHogeveen** [comment]
Sorry, you're right. Are they pulled directly from http://wp-translate.org/projects/pods/ then?

**Comment by sc0ttkclark** [comment]
No, we're no longer using wp-translate, we're relying on the WP core translations.

**Comment by sc0ttkclark** [comment]
I mean, the plugin translations system that's in core, which it downloads the necessary translation files needed from the translate.wordpress.org site

**Comment by JoryHogeveen** [comment]
Are you sure this is configured correctly then? I see two languages available on wp.org (fr_FR and it_IT) but these aren't being pulled from wp.org on an actual installation when I activate these languages.

Sorry for this issue then, I thought since my last conv with @jimtrue this wasn't enabled yet.

**Comment by sc0ttkclark** [comment]
I just pushed some updates, try that and let me know

**Comment by JoryHogeveen** [comment]
Tested it on a fresh WP and Pods install and it works, sorry. No clue why it doesn't download on my test site though, will look into that.

I would revert the changes on `init.php` and `PodsInit.php` for compatibility with manual translation plugins like https://nl.wordpress.org/plugins/loco-translate/
I have the same config on my other plugins and those work fine with wp core aswell.

The other changes apply to the third item right? (#2633)

**Comment by sc0ttkclark** [comment]
Follow up on this, what's left/needed?

**Comment by JoryHogeveen** [comment]
> Call for translation editors for various languages

This is an ongoing process and @jimtrue is very active with it :) Not milestone related.

> Remove auto updates for .po and .mo files from wp.org when ready (#2633)

I believe this is already done??

> Auto update `trunk` on wp.org with the latest dev from the main GH branche

I don't think `trunk is currently being update automaticalle based on 2.x commits right? Not sure if this should be a 2.7 milestone though..
